src/main/java/com/minecolonies/coremod/util/RecipeHandler.java

```diff
@@ -28,6 +28,10 @@ private RecipeHandler()
      */
     public static void init(final boolean enableInDevelopmentFeatures, final boolean supplyChests)
     {
+        GameRegistry.addRecipe(new ItemStack(ModBlocks.blockConstructionTape, 16), "SWS", "S S", "S S",
+          'S', Items.STICK, 'W', new ItemStack(Blocks.WOOL, 1,4));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Assign this magic number 4 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)